FEXCore/Source/Common/SoftFloat.h

```diff
@@ -256,6 +260,12 @@ struct FEX_PACKED X80SoftFloat {
 
     return Result;
 #else
+    // Zero is a special case, the exponent is always -inf
+    if (lhs.Exponent == 0x0 && lhs.Significand == 0x0) {
+      X80SoftFloat Result(1, 0x7FFFUL, 0x8000000000000000UL);
```

Comment:
Hardcoded constants with lots of zeroes like these are prone to typos that aren't easily caught in code review either. There's two ways to avoid this issue:
* Compute the constant symbolically: `1UL << 63`
* Use digit separators to make double-checking easier: `0x8000'0000'0000'0000UL`